core-framework/c-api-framework/include/api/core/Resource.h

```
@@ -131,7 +131,7 @@ void useProcessorClassDescription(Fn&& fn) {
         try {
           static_cast<Class*>(self)->onTrigger(context_wrapper, session_wrapper);
           return MINIFI_SUCCESS;
-        } catch (std::exception& ex) {
+        } catch (std::exception&) {
```

I realize that we should catch all exceptions here "..."

---

and a few lines below this

---

should i update the PR?

---

https://github.com/adamdebreceni/nifi-minifi-cpp/pull/6/commits/8d11d5239af9a5b584a80ca1f7b20c687ea9234a